packages/docs/src/components/page-configs/ui-elements/va-data-table/api-options.ts

```diff
@@ -0,0 +1,36 @@
+import { defineManualApi } from '../../../DocsApi/ManualApiOptions'
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/135523103-2786ec52-0a25-4c42-a3c8-9a3130fc1925.png)

Reply:
Can you check if we can maybe decrease script size by moving data somewhere?

![image](https://user-images.githubusercontent.com/5394573/135523217-87786f72-9c3e-4e76-a0c0-52251a882476.png)

What comes to my mind is some simple API call. You can probably check how it's done for other frameworks. Seems unlikely to me that they include full data in each example.

Reply:
I think it is more convenient for the user to see the data in the example code.
So, to decrease script size I removed redundant data